starter-code/main.html

```diff
@@ -0,0 +1,254 @@
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+        crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy"
+        crossorigin="anonymous">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4"
+        crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+<nav class="container">
+    <div class="top-right">REVERA</div>
+
+    <div class="top-nav">
+        <div>
+            <a href="#">Home</a>
+        </div>
+        <div>
+            <a href="#">About</a>
+        </div>
+        <div>
+            <a href="#">Sample page</a>
+        </div>
+        <div>
+            <a href="#">Portfolio</a>
+        </div>
+        <div>
+            <a href="#">Blog</a>
+        </div>
+        <div>
+            <a href="#">Parent page</a>
+        </div>
+        <div>
+            <a href="#">Contact</a>
+        </div>
+    </div>
+</nav>
+
+<div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img class="d-block w-100" src="./img/slider-1.jpg" alt="First slide">
+        <div class="carousel-caption d-none d-md-block">
+                <h5>Nunc libero elit, iaculis nec augue non</h5>
+                <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur
+                bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur
+                erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse
+                et libero nunc. Etiam mauris tortor, pharetra […]
+                </p>
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="./img/slider-2.jpg" alt="Second slide">
+        <div class="carousel-caption d-none d-md-block">
+                <h5>Phasellus aliquet iaculis neque accumsan aliquam</h5>
+                <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam!
+                Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut
+                magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum
+                bibendum […]
+                </p>
```

Comment:
You have some div tags that are not closing properly. Inspect the elements and you'll see it. Carousel is not appearing because of this.